packages/assets/src/createSpritesheetCli.ts

```diff
@@ -0,0 +1,72 @@
+#! /usr/bin/env node
+/* eslint-disable no-console */
+import { createSpriteSheet } from './createSpritesheet.js';
+import { defaultIcons } from './default-icons.js';
+
+type Args = {
+  icons?: Partial<keyof typeof defaultIcons>[];
+  [key: string]: unknown;
+};
+
+/**
+ * Creates a help message for the CLI.
+ * @returns Help message
+ */
+const helpMessage = (usageError = '') => `Usage: syn-create-spritesheet --icons=icon1,icon2,icon3
+
+Creates a spritesheet string from a list of icons.
+Each icon identifier is a valid icon name for an icon in the synergy icon library.
+
+Using to save the spritesheet string to a file:
+syn-create-spritesheet --icons=a,b,c > icons.svg
+
+Please have a look at https://synergy-design-system.github.io/?path=/docs/icon-search--docs for icon names
+${usageError && `
+Error: ${usageError}`}
+`;
+
+const args = process.argv.slice(2);
+
+// Sanitize the arguments
+const argumentsObject = args.reduce<Args>((acc, arg) => {
+  const [originalKey, originalValue] = arg.trim().split('=');
+
+  const key = originalKey.trim().replace(/^--/, '');
+
+  const finalValue = key === 'icons'
+    ? originalValue.split(',').map((icon) => icon.trim())
+    : originalValue.trim();
+
+  acc[key] = finalValue;
+  return acc;
```

Comment:
I really like the cli! 

But maybe one thing, which would make it a bit more robust, if someone does not know how to use it: 

If using the command with something like this:
```js
pnpm exec syn-create-spritesheet hello
```

following terminal output is thrown:

![image](https://github.com/user-attachments/assets/c64a1bd4-b85c-4798-91a7-f013f4fa2874)


Maybe a try catch could be surrounded and when the catch triggers this is done: 
```js
    console.log(helpMessage());
    process.exit(2);
```

or maybe it is just enough to check for the `originalValue` if is not undefined and if yes the upper code is executed

Reply:
Did not even think about providing bogus values, will add this asap :)

Reply:
Done in latest